idd/Energy+.idd.in

```diff
@@ -40311,10 +40311,32 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
        \note Enter the name of an AvailabilityManagerAssignmentList object.
        \type object-list
        \object-list SystemAvailabilityManagerLists
-  A16; \field Design Specification ZoneHVAC Sizing Object Name
+  A16, \field Design Specification ZoneHVAC Sizing Object Name
        \note Enter the name of a DesignSpecificationZoneHVACSizing object.
        \type object-list
        \object-list DesignSpecificationZoneHVACSizingName
+  A17, \field Supplemental Heating Coil Object Type
+       \type choice
+       \key Coil:Heating:Fuel
+       \key Coil:Heating:Electric
+       \key Coil:Heating:Water
+       \key Coil:Heating:Steam
+       \note works with gas, electric, hot water and steam heating coil.
+  A18, \field Supplemental Heating Coil Name
+       \type object-list
+       \object-list HeatingCoilName
+       \note Needs to match in the supplemental heating coil object. 
+  N11, \field Maximum Supply Air Temperature from Supplemental Heater
+       \type real
+       \units C
+       \autosizable   
```

Comment:
This should have \default autosize and that should be mentioned in the I/O Ref (same for the other one).

Reply:
Updated the IO reference of the WSHP and VRF terminal units.